bionty/__init__.py

```diff
@@ -68,10 +68,18 @@
 
    Read the guides:
 
-   - `Access public ontologies <https://lamin.ai/docs/public-ontologies>`__
+   - :doc:`docs:public-ontologies`
    - :doc:`docs:bio-registries`
 
-   For more background on how public ontologies are accessed, see `bionty.base`.
+Bionty base
+===========
+
+Bionty also offers a standalone version `bionty-base` that does not require an active lamindb instance.
+Import it by running:
+
+>>> import bionty.base as bt_base
+
+See {doc}`docs:bionty.base` for details.
```

Comment:
See the suggestion of above - no intersphinx needed; and it's a module, not a doc.
<img width="148" alt="image" src="https://github.com/user-attachments/assets/80f06898-a252-4d94-9713-0e23dbd8a1fa">